### PR TITLE
Adds Bellatrix date warning

### PR DESCRIPTION
**Pull request by wackerow**
image

The last update to the Merge Readiness page included the TTD and estimated date for Mainnet merge, but did not include a note about the timing of the required Bellatrix upgrade set to occur for the Beacon Chain at epoch 144896. 

- Adds note about upgrading before Bellatrix
- Adds intl support for date/time formatting so these display appropriately in all locales.

### CODE REVIEW

**Review by CarlBeek — APPROVED** [review]
Thanks @wackerow, this is a helpful warning & the numbers check out.

One thing I think we should think about is the president of reminding users of upcoming upgrades via the launchpad. Is this something we want to do and maintain moving forward?

File: src/pages/MergeReadiness/index.tsx [hunk 1]
```diff
@@ -148,47 +148,90 @@ export const MergeReadiness = () => {
...
+          <Text className="mb20">
+            <FormattedMessage
+              defaultMessage="The Bellatrix upgrade to the Beacon Chain is scheduled for epoch 144896, set to occur at {bellatrixDateTime}."
+              values={{ bellatrixDateTime }}
+            />
+          </Text>
+          <Text className="mb20">
+            <FormattedMessage
+              defaultMessage="Mainnet total terminal difficulty (TTD) has been set to {difficulty}. Ethereum is estimated to reach this TTD within a day of {ttdDateTime} (follow {link} for latest estimate)."
+              values={{
```
[the remark this answers is not in the document]
Comment (wackerow) [reply src/pages/MergeReadiness/index.tsx:195]: Right, this was the main reason for adding the note about being ready by Sept 6 in this PR (just below this sentence)

Bordel estimates for when we hit TTD haven't changed much, but perhaps we could adjust this to "within a few days of" to give a little more wiggle room on the TTD, but the main deadline for node operators is the 6th... personally not so worried about the exact crossing of the TTD.

**Comment by wackerow** [comment]
New additions:
- Added the date range from 9/10 to 9/20 for consistency with other EF communications
- Added link to 797.io/themerge as a second TTD tracker
- Added date information to "When Merge?" bonus section

image

image

**Review by hwwhww — APPROVED** [review]
lgtm 👍👍

**Pull request merged**